D/D.sublime-syntax

```diff
@@ -1191,12 +1198,16 @@ contexts:
         - match: '\('
           scope: punctuation.section.parens.begin.d
           set:
-            - match: '\b{{name}}\b'
-              scope: variable.function.d
+            - match: '\b({{traits}})\b'
+              scope: constant.other.d
               set:
                 - match: ','
                   scope: punctuation.separator.sequence.d
                   set: [value-parens-after, value-list]
+                - match: '\)'
+                  scope: punctuation.section.parens.end.d
+                  pop: true
+            - include: not-whitespace-illegal-pop
```

Comment:
As @deathaxe mentioned, I think it would be preferrable if this didn't assign an illegal scope but rather a generic scope. That would still differ from the known-good values matched via `{{traits}}` but not mark anything else as illegal, which is especially annoying during typing.

Reply:
Feel free to reopen it then

Reply:
Right, I somehow though the branch was deleted, in which case a PR cannot be re-opened.